vendor/src/github.com/docker/engine-api/types/events/events.go

```diff
@@ -9,8 +9,14 @@ const (
 	VolumeEventType = "volume"
 	// NetworkEventType is the event type that networks generate
 	NetworkEventType = "network"
-	// DaemonEventType is the event type that daemon generate
+	// DaemonEventType is the event type that daemon generates
 	DaemonEventType = "daemon"
+	// SwarmEventType is the event type that swarm generates
+	SwarmEventType = "swarm"
+	// NodeEventType is the event type that nodes generate
+	NodeEventType = "node"
+	// ServiceEventType is the event type that services generate
+	ServiceEventType = "service"
```

Comment:
Do you plan to publish all events related to a service? What i was hoping for was the ability to stream all events related to a service (create/update/containers kill/start/health change/etc). If there are 2 types (container & service) then the current way events are filtered won't be able to do show both.